update recent history (#289)
better-internet/radworks-docs: docs/community/02_our_story.md

  desc="[Radicle](https://boardroom.io/radworks/proposal/cHJvcG9zYWw6cmFkaWNsZTpvbmNoYWluOjE0) and [Drips](https://boardroom.io/radworks/proposal/cHJvcG9zYWw6cmFkaWNsZTpvbmNoYWluOjE1) and [Grants](https://boardroom.io/radworks/proposal/cHJvcG9zYWw6cmFkaWNsZTpzbmFwc2hvdDoweDgwYTFmOTJmOTVmMzQ4ZDYwZWVhYzE2MjcyYmFlOTQwY2ExYjkwNDc4ZjcwYTcyY2YzNDI2M2Y1Y2Y0OGMyYmI=) became independent, DAO-funded Orgs within the DAO and the [Org Design Working Group](https://community.radworks.org/t/open-call-core-development-org-design-working-group/3042/3), which led the charge of DAO transition design & research, was sunset."
/>

<Timeline
  date="July, 2023"
  title="RadicleDAO becomes [Radworks](https://radworks.mirror.xyz/PHFoG1xR98l2eZhkgr4281AaxbXnHD2iVGhip6zbmSw)"
  desc="The DAO identity shifts to be more inclusive for all technology being supported by this community-governed network."
/>

<Timeline
  date="July, 2023"
  title="[Drips V2 launches](https://x.com/dripsnetwork/status/1684190676332822530?s=20)"

>   desc="Intoduces a new platform, Drips lists, oracle-based identity solution & xxx"

Reviewer: ```suggestion
  desc="Introduces Drips lists, an oracle-based identity solution, and a new app."
```
